libraries/c_sdk/standard/mqtt/test/iot_test_mqtt_agent.c

```diff
@@ -125,13 +125,13 @@
 
 
 /* Default connection parameters. */
```

Comment:
Context for reviewers: This change is needed so that the thing name can be retrieved via function.